plugins/pulp_ostree/plugins/distributors/configuration.py

```diff
@@ -112,7 +112,8 @@ def _check_for_relative_path_conflicts(repo_id, relative_path, conduit):
     :rtype: list
     """
     messages = []
-    distributors = conduit.get_repo_distributors_by_relative_url(relative_path, repo_id)
+    distributors = conduit.get_repo_distributors_by_relative_url(
+        relative_path, repo_id=repo_id, field='relative_path')
     for distributor in distributors:
         conflicting_repo_id = distributor['repo_id']
         conflicting_relative_url = distributor['config']['relative_path']
```

Comment:
no reason at all, that was just first thing came to my mind, but your's is much better! :star2: